utils/lib/connection_database.sql

```diff
@@ -81,6 +81,7 @@ CREATE TABLE phy_tile(
   grid_x INT,
   grid_y INT,
   clock_region_pkey INT,
+  prohibited INT,
```

Comment:
Sites are prohibited, not tiles.  This is the wrong place for the flag.  Also this should be a BOOLEAN, not an INT.

Reply:
The only reason to set the prohibited to Tiles is that it was more straightforward to skip the tile itself during arch_import, but I guess it is more consistent as you suggested. Will fix that and bump prjxray

Reply:
Fixed